<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReplayWindow</class>
 <widget class="QMainWindow" name="ReplayWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Replay</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="turnLabel">
      <property name="text">
       <string>현재 턴:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 16px; font-weight: bold;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="countLabel">
      <property name="text">
       <string>흑: 0, 백: 0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="boardWidget" native="true"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="navigationLayout">
      <item>
       <widget class="QPushButton" name="prevButton">
        <property name="text">
         <string>◀</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="turnSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="nextButton">
        <property name="text">
         <string>▶</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="bottomLayout">
      <item>
       <spacer name="spacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="mainMenuButton">
        <property name="text">
         <string>메인화면</string>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>30</height>
         </size>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum"/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar"/>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
